endpoints/data_objects/src/main.cpp

```diff
@@ -357,6 +357,124 @@ namespace
 		std::int64_t remaining_bytes_;
 	}; // incremental_read
 
+	class incremental_write : public std::enable_shared_from_this<incremental_write>
+	{
+	  public:
+		incremental_write(
+			irods::http::session_pointer_type& _sess_ptr,
+			unsigned int _http_version,
+			bool _http_keep_alive,
+			irods::http::connection_facade& _conn,
+			std::unique_ptr<io::client::native_transport>& _tp,
+			std::unique_ptr<io::odstream>& _out,
+			io::odstream* _out_ptr,
+			std::unique_ptr<irods::at_scope_exit<std::function<void()>>>& _mark_pw_stream_as_usable,
+			std::string& _buffer,
```

Comment:
Could make it more obvious that ownership of some inputs is being transferred by requiring an R-value.

Making that change would only affect the call site. There would be no change in behavior.

Reply:
Done.